.swiftpm/xcode/xcuserdata/markwebb.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>DeviceKitPlayground (Playground) 1.xcscheme</key>
```

Comment:
What are the various playgrounds for?

Reply:
This should probz be .gitignored as it's xcuserdata! Good spot :D